src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  height: 408px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__product-code {
+    align-self: start;
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text-color;
+  }
+
+  &__reviews {
+    margin-top: 16px;
+  }
+
+  &__price {
+    margin-top: 24px;
+  }
+
+  &__link {
+    width: 100%;
+    height: 40px;
+    margin-top: 16px;
+    border: solid 0;
+    border-radius: 5px;
+    background-color: #00ACDC;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16;
```

Comment:
The line-height property should have a unit. It appears you forgot to include 'px' in the line-height. Change all cases!